src/main/java/staffconnect/model/meeting/Description.java

```diff
@@ -0,0 +1,66 @@
+package staffconnect.model.meeting;
+
+import static java.util.Objects.requireNonNull;
+import static staffconnect.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Meeting's description in the staff book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidDescription(String)}
+ */
+public class Description {
+
+    public static final String MESSAGE_CONSTRAINTS =
+        "Description should only contain alphanumeric characters and spaces, and it should not be blank";
+
+    /*
+     * The first character of the name must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+
+    public final String description;
+
+    /**
+     * Constructs a {@code Description}.
+     *
+     * @param textDescription A valid name.
```

Comment:
I think you missed this part out. Don't need to commit this suggestion here, you can do it in your local repository.
```suggestion
     * @param textDescription A valid description.
```